content/guides/concepts/dht.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Distributed Hash Table (DHT)"
+menu:
+    guides:
+        parent: concepts
+---
+
+# DHTs
+
+## What is a DHT?
+
+[Distributed Hash Tables](https://en.wikipedia.org/wiki/Distributed_hash_table) (DHT) are a distributed key-value store where keys are [cryptographic hashes](https://docs.ipfs.io/guides/concepts/hashes/). 
+
+DHTs are distributed. Each "peer" (or "node") is responsible for a subset of the DHT. 
+A receiving peer either answers the request, or the request is passed to another peer until a peer can answer it.
+Depending on implementations, a request not answered by the first contacted node can be :
+- forwarded from peer to peer and the last peer contact the requesting peer
+- forwarded from peer to peer and the answered is forwarded following the same path
+- answered with contacting information of a node having better chances to be able to answer (IPFS uses this strategy)
+
+DHT's decentralization provides advantages compared to a classic Key-Value store:
+- *scalability* as a request for a hash of length *n* takes at most *log2(n)* steps to resolve.
+- *fault tolerance* via redundancy, so that lookups are possible even if peers unexpectedly leave or join the DHT. Additionally, requests can be addressed to any peer if one is slow or unavailable.
```

Comment:
```suggestion
- *fault tolerance* via redundancy, so that lookups are possible even if peers unexpectedly leave or join the DHT. Additionally, requests can be addressed to any peer if another peer is slow or unavailable.
```

Reply:
Moved this change to https://github.com/ipfs/docs/pull/364